libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/testbase/BuildOptions.kt

```diff
@@ -133,6 +135,14 @@ data class BuildOptions(
         if (usePreciseJavaTracking != null) {
             arguments.add("-Pkotlin.incremental.usePreciseJavaTracking=$usePreciseJavaTracking")
         }
+
+        if (debug) {
```

Comment:
There is already enableGradleDebug on `TestProject.build` method. Do we need an additional one?

Reply:
Nice, I didn't know about that. I've reverted this change and add `kotlinDaemonDebugPort` to `TestProject.build`.